### ISSUE TITLE
Unable to serve Snipe-IT (Kubernetes) over HTTPS, APP_TRUSTED_PROXIES ignored

### ISSUE THREAD

**Issue by nuraeil** [grokability/snipe-it#9581]
#### Please confirm you have done the following before posting your bug report:

- [X] I have enabled debug mode 
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**

I am referencing to an issue from over 3 years ago: #6089 

I am currently unable to setup Snipe-IT in our Kubernetes environment over HTTPS. I followed every available ressource to try and configure it but to no avail.

Setting APP_TRUSTED_PROXIES to our Nginx Ingress Endpoint or to '*' does nothing. The APP_URL is also set to https://[URL]:443. 

When trying to configure SSL inside the container itself, the container does not even expose port 443 and simply disables SSL. 

`Module ssl disabled.
To activate the new configuration, you need to run:
  service apache2 restart`

![image](https://user-images.githubusercontent.com/32387849/118259992-15ed8c80-b4b2-11eb-858b-873d66832f7a.png)

![image](https://user-images.githubusercontent.com/32387849/118259859-f6eefa80-b4b1-11eb-821d-481c10b5ab04.png)

I even tried setting everything manually and the result is that the Apache, inside the container, thinks that I am accessing it over HTTP even though I am accessing it over HTTPS...

![chrome_2021-05-14_13-03-47](https://user-images.githubusercontent.com/32387849/118262282-f441d480-b4b4-11eb-9c84-4d2c657fa821.png)

![image](https://user-images.githubusercontent.com/32387849/118264450-007b6100-b4b8-11eb-9b00-889bc9838850.png)

- I integrated the generated certificates from Let's Encrypt into the container
- I manually changed the SSL configuration from Apache in order to be able to use the certficiates as they were read-only (simply pointed to the correct certificates)
- I manually enabled (`a2enmod ssl`) SSL and reloaded
- Manually changed the Load Balancer to point to port 443 rather than 80
- Manually changed the .htaccess file according to the documentation (https://snipe-it.readme.io/docs/configuration)

I even tried enforcing certain rules from the Nginx Ingress itself, that it force redirects all traffic to HTTPS and that the Backend communicates over HTTPS, nothing.

When I point the Nginx Ingress to port 80, the site loads but I have no CSS or anything as the app tries to access the ressources  over HTTP.

![chrome_2021-05-14_13-19-55](https://user-images.githubusercontent.com/32387849/118263888-2c4a1700-b4b7-11eb-9c28-ffca2c280cbb.png)

**To Reproduce**
1. Configure an environment for Snipe-IT in Kubernetes with an Nginx Ingress
2. Try to expose it over HTTPS through Nginx Ingress

**Expected behavior**
The site and all the ressources should be served over HTTPS and not HTTP.

**Screenshots**
In above explanation.

**Server (please complete the following information):**
 - Snipe-IT Version: 5.1.5 
 - Kubernetes/Docker
 - Nginx Ingress

**Error Messages**
![chrome_2021-05-14_13-03-47](https://user-images.githubusercontent.com/32387849/118262282-f441d480-b4b4-11eb-9c84-4d2c657fa821.png)
![chrome_2021-05-14_13-19-55](https://user-images.githubusercontent.com/32387849/118263888-2c4a1700-b4b7-11eb-9c28-ffca2c280cbb.png)

**Additional context**
- It is a fresh install.
- Certificates are from Let's Encrypt through cert-issuer
- The Database is a MySQL 8.X instance.
- Kubernetes Cluster with a Rancher as Frontend and NGINX for serving sites.
- Docker Image from Dockerhub and also a different image from linuxserver/snipe-it
- ENV set that were recommended or nececessary

**Comment by nuraeil** [comment]
So I found #6852 ... and I manually edited the trustedproxy.php and set proxy to *... now it works... But that should not be the way to "fix" this as the file even mentiones that the settings are pulled from the supplied envs.

I also had to set it to * rather than the IP address of my Nginx Proxy... because the IP did not seem to be accepted. 

It does not seem to correctly get the env variable... or rather it has it set somewhere to something I can not discern what it excatly is set to.

**Comment by nuraeil** [comment]
I am going to close this issue as I simply did not have a correct overview of our current server architecture.

Our internal setup is not very well documented in terms of what system is involved in the communication process and pointing it only to our NGINX Ingress IP was not enough... 

I had to put around 5 other of our IP addresses into the APP_TRUSTED_PROXIES in order to make that work.

In order to find out which IP addresses were used, I used the Laravel debugger.

The env is now unset and it works fine.

I am sorry for the trouble!
